online-courses.md

```diff
@@ -20,21 +20,17 @@ testimonials:
   - /resources/images/online-courses/testimonial014.jpg
 ---
 
-The online courses from the [Scala Center] provide two main paths to learn 
-Scala. The fast path consists of taking the course [Effective Programming 
-in Scala],
-otherwise you can take the full [Scala Specialization], which is made of
-four courses and a capstone project.
+The [Scala Center] produces online courses of various levels, from beginner 
+to advanced.
 
-All the courses are available for free. Optionally, a subscription gives
-you access to a certificate of completion that attests your accomplishments.
-Learn more about
-[Coursera certificates](https://learners.coursera.help/hc/en-us/articles/209819053-Get-a-Course-Certificate) or
-[edX certificates](https://support.edx.org/hc/en-us/categories/115002269627-Certificates).
-Your subscriptions also supports the work of the [Scala Center], whose mission
-is to create high-quality educational material.
+If you are a programmer and want to learn Scala, there are two recommended 
```

Comment:
You could maybe put this part "if you are a programmer and want to learn Scala" in bold ? So that people who are skim reading can catch on it.